src/main/java/cloud/eppo/ufc/dto/Attributes.java

```diff
@@ -0,0 +1,64 @@
+package cloud.eppo.ufc.dto;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Attributes extends HashMap<String, EppoValue> implements DiscriminableAttributes {
```

Comment:
This is a rename from `SubjectAttributes`, but we've added enough new stuff that Git is confused.

This and `ContextAttributes` implement `DiscriminableAttributes`, the difference is the former has the numeric vs. categorical type explicitly specified.